src/lib/annotations/Annotator.js

```diff
@@ -541,25 +563,66 @@ class Annotator extends EventEmitter {
     }
 
     /**
-     * Binds event listeners for point annotation mode.
+     * Binds event listeners for annotation modes.
      *
      * @protected
+     * @param {string} mode - Current annotation mode
      * @return {void}
      */
-    bindPointModeListeners() {
-        const pointFunc = this.pointClickHandler.bind(this.annotatedElement);
-        const handlers = [
-            {
-                type: 'mousedown',
-                func: pointFunc,
-                eventObj: this.annotatedElement
-            },
-            {
-                type: 'touchstart',
-                func: pointFunc,
-                eventObj: this.annotatedElement
+    bindModeListeners(mode) {
+        const handlers = [];
+        if (mode === TYPES.point) {
+            const pointFunc = this.pointClickHandler.bind(this.annotatedElement);
+            handlers.extend([
```

Comment:
My mistake on this syntax, extend is a python function.
Looks like we need to use a spread operator: `handlers.push(...[h1,h2,h3])`